### PR TITLE
Avocado New Website Master Merge

### PR DESCRIPTION
**Pull request by harvey0100**
Avocado website updated according to designs provided lots of new content such as updated utils table.
Many changes to the Main Avocado Framework Page included. Other repos such as Aexpect / VT links point to their repo.

Website Requested Changes

Added License, Credits to designers.
Deployment script added for workflow to
fetch avatars for the autils table.

-- Layout on framework page is now matching that of home requested by @richtja

### CODE REVIEW

**Comment by richtja** [comment]
Hi @harvey0100, thank you for moving this to master branch, Can you please run the build in your fork with the latest changes and share the results. Thanks

**Comment by harvey0100** [comment]
https://github.com/harvey0100/harvey0100.github.io/actions/runs/11627808952 - latest workflow working and built succeeding harvey0100.github.io can be viewed here, i believe this is ready for deployment.

File: .github/workflows/jekyll-gh-pages-avatar.yml
Comment (richtja) [inline .github/workflows/jekyll-gh-pages-avatar.yml:42]: This command is focused only to archive utility, when we will add more utils to autils this command won't work. This needs to use `ls -R resources/autils/metadata/autils/`

File: .github/workflows/jekyll-gh-pages-avatar.yml
Comment (richtja) [inline .github/workflows/jekyll-gh-pages-avatar.yml:48]: same here.

**Review by richtja — CHANGES_REQUESTED** [review]
Hi @harvey0100, thanks for the updates. I have one more comment to the deployment workflow, please let me what do you think. 

Also, I have checked the website itself, and it looks like there are still some testing data [there](https://harvey0100.github.io/autils.html). IMO, the `Alice` and `Bob` shouldn't be part of this table.

File: .github/workflows/jekyll-gh-pages-avatar.yml
Comment (harvey0100) [reply .github/workflows/jekyll-gh-pages-avatar.yml]: Hi Jan, changed to your request :)

File: .github/workflows/jekyll-gh-pages-avatar.yml
Comment (richtja) [inline .github/workflows/jekyll-gh-pages-avatar.yml:48]: Have you test it out in the workflow? I am not sure what are the shell capabilities in the GitHub workflow, therefore I am asking. If it is working, I am fine with it.

File: .github/workflows/jekyll-gh-pages-avatar.yml
Comment (richtja) [reply .github/workflows/jekyll-gh-pages-avatar.yml]: I see, thanks for the explanation.

File: .github/workflows/jekyll-gh-pages-avatar.yml
Comment (harvey0100) [reply .github/workflows/jekyll-gh-pages-avatar.yml:48]: Tested in my fork and workflow, build is still completing and everything still working on website :)

**Review by richtja — COMMENTED** [review]
Beside my last comment it LGTM, but before the merge I would wait for the @clebergnu review since this completely changes our website, therefore I would like to see more reviews here. And if everything will be ok, I would propose to wait with the merge until Monday, so we would have time to solve any post merge issues if they occur.

File: .github/workflows/jekyll-gh-pages-avatar.yml
Comment (richtja) [reply .github/workflows/jekyll-gh-pages-avatar.yml:48]: ok great

**Review by richtja — APPROVED** [review]

**Pull request merged**
